pkg/cluster/internal/create/createcontext.go

```diff
@@ -146,6 +146,12 @@ func (cc *Context) ProvisionNodes() (nodeList map[string]*nodes.Node, err error)
 			return nodeList, err
 		}
 
+		cc.Status.Start(fmt.Sprintf("[%s] Configuring proxy 🐋", configNode.Name))
```

Comment:
had to view this RAW and search the U+... in google as none of my fonts support this glyph.
i think we need a different icon for the proxy and not a "whale".

also my suggestion to move these to constants was not accepted. :grumble:

Reply:
It worked just fine fine for me in Mac Terminal. As for another icon, so far the code configures proxy for Docker, so I decided to use "whale" glyph.

Can you suggest where can I see a list of supported glyphs?

Reply:
i guess the whale also makes sense. i'm going to leave this topic to @BenTheElder

Reply:
This is fine for now and on par with the rest. I'll probably revisit this UX late next week. Output needs cleanuo.

Reply:
I'm not sure if we need a status for this step given how quick it should be, but like I said we need to look at cleaning this up anyhow.

Aside: what windows are you using Lubomir? Windows should have emoji fonts and even has some custom nonstandard emoji.